frontend/src/components/dashboard/JoinOrganizationModal.tsx

```diff
@@ -347,7 +347,7 @@ function JoinOrganizationForm({
                   />
                   <div className="pb-1 text-sm text-[#6B7280]">
                     <label htmlFor="confirmationWorkingForTheOrg">
-                      I confirm that I work for this organisation
+                      I confirm that I am a member of this organization.
```

Comment:
@khaeru we used British English everywhere else, so I just want to confirm you are aware of that before we merge this one i.e. this would be the only textual content in the UI in which "organization" is written this way (afaik).

Reply:
> we used British English everywhere else

Okay, thanks for clarifying. I'll update the branch for consistency.

Is that written somewhere? I looked in e.g. "technical documentation" [here](https://github.com/transport-data/tdc-data-portal/tree/main/docs) but did not find any info along those lines.

Reply:
@khaeru 

No, I don't think this is documented anywhere, but we did it this way based on the Figma mock-ups